docs/sources/clients/promtail/configuration.md

```diff
@@ -95,6 +95,9 @@ clients:
 scrape_configs:
   - [<scrape_config>]
 
+# Configures global limits for this instance of Promtail
+[limit_config: <limit_config>]
```

Comment:
hi, thank you very much for the exquisite documentation, this PR changed the limit_config name

https://github.com/grafana/loki/pull/5707
Promtail: Rename config name limit_config to limits_config #5707